README-development.md

```diff
@@ -1,3 +1,49 @@
+# Operating System
+
+## Prerequisites
+
+### Usually packaged by operating systems
+- git
+- ostree
+- rpm-ostree
+- qemu-img
+- fedpkg
+- mock
+
+### May need to be built from source
+- [coreos-assembler](https://github.com/cgwalters/coreos-assembler/): [container](http://quay.io/cgwalters/coreos-assembler)
```

Comment:
Note https://github.com/openshift/os/pull/119 should make this unnecessary.

Reply:
Is `coreos-assembler` deprecated at this point as well?